taf/docker-compose-taf-perf-arm64.yml

```diff
@@ -50,13 +50,13 @@ services:
       EDGEX_PROFILE: mqtt-export
       EDGEX_SECURITY_SECRET_STORE: "false"
       REGISTRY_HOST: edgex-core-consul
-      SERVICE_HOST: app-service-mqtt-export
+      SERVICE_HOST: app-mqtt-export
```

Comment:
Looks like the security portions are missing here.

Reply:
good catch, i missed it!

Reply:
fixed